Source/Charts/Charts/CombinedChartView.swift

```diff
@@ -120,7 +120,7 @@ open class CombinedChartView: BarLineChartViewBase, CombinedChartDataProvider
             {
                 return nil
             }
-            return (_data as! CombinedChartData!).lineData
```

Comment:
It has been sometime we review this part. But seems `CombinedChartData!` can be `CombinedChartData` now,

Reply:
updated